### PR TITLE
Bug/clear search history on workspace switch

### PR DESCRIPTION
**Pull request by paul-ciobanu**

### CODE REVIEW

File: OctaneVSPlugin/View/PluginWindowManager.cs [hunk 1]
```diff
@@ -51,6 +51,9 @@ internal static DetailsToolWindow ObtainDetailsWindow(MainWindowPackage package,
         /// </summary>
         internal static SearchToolWindow ObtainSearchWindow(MainWindowPackage package)
         {
+            if (package == null)
```
Comment (atothhpe) [inline OctaneVSPlugin/View/PluginWindowManager.cs]: ternary operator ? it's your style choice though

File: octane-visual-studio-plugin-tests/Common/SearchHistoryManagerTests.cs [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+            CollectionAssert.AreEqual(new List<string>(), SearchHistoryManager.History, "Mismatched history after reset");
+
+            SearchHistoryManager.UpdateHistory("2");
+
+            CollectionAssert.AreEqual(new List<string> { "2" }, SearchHistoryManager.History, "Mismatched history");
+        }
+
+
+        // save, _metadata = null, (restart), check history, save check history
```
Comment (atothhpe) [inline octane-visual-studio-plugin-tests/Common/SearchHistoryManagerTests.cs]: Are these comments supposed to be here?

Comment (paul-ciobanu) [reply octane-visual-studio-plugin-tests/Common/SearchHistoryManagerTests.cs]: @atothhpe removed

File: OctaneVSPlugin/View/PluginWindowManager.cs (hunk 1 above)
Comment (paul-ciobanu) [reply OctaneVSPlugin/View/PluginWindowManager.cs]: @atothhpe done

**Pull request merged**